paper.tex

```diff
@@ -495,6 +527,13 @@ \subsection{Modeling Effects}
     N_{wasted}(t) = \sum\limits_{r \in R_t} \left(H[t-S_{sched}(t,r)] \cdot [1-O(t,r)]  \sum\limits_{t'=F_{prev}(t,r)}^{t} N_b(t',r) \right)
     \label{eq:badshare}
 \end{equation}
+%% What is the value of this function during a normal refueling outage,
+%% immediately before S_(sched)(t)?  It seems you need another Heavyside
+%% function to drop this back down to an expected value of 0 again.
```

Comment:
That is what ``O`` does - or am I misunderstanding your comment?  If the reactor is operating (i.e. at the end of a normal outage), ``O`` becomes zero.